schemas/JSON/manifests/v1.2.0/manifest.locale.1.2.0.json

```
@@ -35,6 +35,21 @@
           "description": "The agreement URL."
         }
       }
+    },
+    "Documentation": {
+      "type": "object",
+      "properties": {
+        "DocumentLabel": {
+          "type": [ "string", "null" ],

```

Documents should always be labeled
```suggestion
          "type": "string",
```